03/app.js

```diff
@@ -42,11 +42,25 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    if (this[isbn] === 'undefined') {
```

Comment:
Tutaj bardzo trudno jest zrozumieć mentee dlaczego potrzeba sprawdzać "każdy" klucz czy istnieje - po prostu nie wiemy czy kolejne zagnieżdżenie posiada odpowiedni typ wartości (obiekt/tablicę) i daną właściwość. Jeśli nie i tego nie sprawdzimy to będzie błąd czyli program przestanie działać.